src/components/generics/back-button.tsx

```diff
@@ -12,41 +11,40 @@ interface BackButtonProps {
 }
 
 /**
- * Returns the localized "Back" label for a given module key.
+ * Returns the localized "Back" label for a given module key 
+ * which also allows the translation to function properly.
  *
  * @param moduleKey - A key from the `strings` object or null
  */
 const getBackLabel = (moduleKey: keyof typeof strings | null): string => {
   if (!moduleKey) return "Back";
```

Comment:
I'm still unclear why we are returning non localized strings. All strings should be localized?